CIP-meta-assets (ERC20-like assets)/README.md

```diff
@@ -0,0 +1,448 @@
+---
+CIP: _???
+Title: meta-assets (ERC20-like assets)
+Category: Tokens
+Status: Proposed
+Authors:
+    - Michele Nuzzi <michele.nuzzi.204@gmail.com>
+    - Harmonic Laboratories <harmoniclabs@protonmail.com>
+    - Matteo Coppola <m.coppola.mazzetti@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2023-01-14
+License: CC-BY-4.0
+---
+
+<!-- Existing categories:
+
+- Meta                   | For meta-CIPs which typically serves another category or group of categories.
+- Reward-Sharing Schemes | For CIPs discussing the reward & incentive mechanisms of the protocol.
+- Wallets                | For standardisation across wallets (hardware, full-node or light).
+- Tokens                 | About tokens (fungible or non-fungible) and minting policies in general.
+- Metadata               | For proposals around metadata (on-chain or off-chain).
+- Tools                  | A broad category for ecosystem tools not falling into any other category.
+- Plutus                 | Changes or additions to Plutus
+- Ledger                 | For proposals regarding the Cardano ledger
+- Catalyst               | For proposals affecting Project Catalyst / the Jörmungandr project
+
+-->
+
+# CIP-XXXX: meta-assets (ERC20-like assets)
+<!-- A reminder of the document's number & title, for readability. -->
+
+## Abstract
+<!-- A short (\~200 word) description of the proposed solution and the technical issue being addressed. -->
+This CIP proposes a standard that if adopted would allow the same level of programmability of other ecosistems at the price of the token true ownership.
+
+This is achieved imitating the way the account model works laveraging the UTxO structure adopted by Cardano.
+
+## Motivation: why is this CIP necessary?
+<!-- A clear explanation that introduces the reason for a proposal, its use cases and stakeholders. If the CIP changes an established design then it must outline design issues that motivate a rework. For complex proposals, authors must write a Cardano Problem Statement (CPS) as defined in CIP-9999 and link to it as the `Motivation`. -->
```

Comment:
```suggestion
```
Sorry I should have noted how many of these template comments there were & simply told _you_ to get rid of them 😅